app/models/organisation.rb

```diff
@@ -1,11 +1,11 @@
 class Organisation < ApplicationRecord
-  has_many :users
-  has_many :owned_case_logs, class_name: "CaseLog", foreign_key: "owning_organisation_id"
-  has_many :managed_case_logs, class_name: "CaseLog", foreign_key: "managing_organisation_id"
+  has_many :users, dependent: :destroy
+  has_many :owned_case_logs, class_name: "CaseLog", foreign_key: "owning_organisation_id", dependent: :destroy
+  has_many :managed_case_logs, class_name: "CaseLog", foreign_key: "managing_organisation_id", dependent: :destroy
```

Comment:
Don't think we want it here. I'm not sure deleting a managing organisation implies the owning organisation's logs should be deleted.

Reply:
I have deleted the "dependent: :destroy" and pushed again.